intuita-webview/src/codemodList/TreeView/index.tsx

```diff
@@ -14,14 +14,44 @@ import styles from './style.module.css';
 import cn from 'classnames';
 import { useProgressBar } from '../useProgressBar';
 import { VSCodeButton } from '@vscode/webview-ui-toolkit/react';
+import { SEARCH_QUERY_MIN_LENGTH } from '../../shared/SearchBar';
 
 type Props = Readonly<{
 	node: CodemodTreeNode;
 	autocompleteItems: string[];
 	openedIds: ReadonlySet<CodemodHash>;
 	focusedId: CodemodHash | null;
+	searchQuery: string;
 }>;
 
+export const containsSearchedCodemod = (
+	node: CodemodTreeNode,
+	searchQuery: string,
+	set: Set<CodemodHash>,
+): boolean => {
+	if (
+		node.kind === 'codemodItem' &&
+		(node.uri.toLowerCase().includes(searchQuery) ||
```

Comment:
i'd convert searchQuery to lowerCase and trim it as well 
`searchQuery.toLowerCase().trim()`